src/script/cpatch

```diff
@@ -30,7 +30,7 @@ usage() {
     echo "  --push               push when done"
     echo "  --strip              strip binaries"
     echo "  --root-build         build image as root"
-    echo
+    echo "  --noall              skip all other operations and just check if build image is available"
```

Comment:
how does this script "check if build image is available"? could you paste a sample output here?